pkg/pillar/types/global.go

```diff
@@ -166,6 +166,7 @@ const (
 	// MintimeUpdateSuccess global setting key
 	MintimeUpdateSuccess GlobalSettingKey = "timer.test.baseimage.update"
 	// StaleConfigTime global setting key
+	// DEPRECATED! Saved config never expires!
 	StaleConfigTime GlobalSettingKey = "timer.use.config.checkpoint"
```

Comment:
Should we not remove this (including in the docs markdown file)?

I think it would be confusing to keep it if it has no effect.

In any case we need to communicate to users that this is gone and the time is effectively infinite.

Reply:
There is some logic behind config merges after/during upgrade. I'm not touching this in order not to break things. Or this is safe to remove completely? 

Docs I could miss. Will check that.